### PR TITLE
Allow users to modify event slugs

### PR DESCRIPTION
**Pull request by tmb**

### CODE REVIEW

File: app/views/events/_form.html.erb [hunk 1]
```diff
@@ -16,6 +16,11 @@
     <%= form.text_area :address %>
   </div>
 
+  <div class="field">
+    <%= form.label :slug %>
+    <%= form.text_field :slug %>
+  </div>
+
```
Comment (lachlanjc) [inline app/views/events/_form.html.erb:36]:
```suggestion
```

**Review by lachlanjc — CHANGES_REQUESTED** [review]
Looks great, let’s just add an explanation of what a slug is.

File: app/views/events/_form.html.erb [hunk 2]
```diff
@@ -16,6 +16,10 @@
     <%= form.text_area :address %>
   </div>
 
+  <div class="field">
+    <%= form.label :slug, 'Event URL' %>
+    <%= form.text_field :slug %>
```
Comment (lachlanjc) [inline app/views/events/_form.html.erb]: I would put my event website URL in this field even if I wrote this form code myself. (Sidenote, does typing a URL in here break anything?) Let’s switch back to “slug” & add helper text explaining what this is. For styling, use: https://github.com/hackclub/bank/blob/master/app/views/g_suite_accounts/_new.html.erb#L35

**Review by lachlanjc — CHANGES_REQUESTED** [review]

File: app/controllers/events_controller.rb [hunk 3]
```diff
@@ -89,6 +89,9 @@ def g_suite_overview
   # Use callbacks to share common setup or constraints between actions.
   def set_event
     @event = Event.find(params[:id])
+  rescue ActiveRecord::RecordNotFound
+      flash[:error] = "We couldn't find that event!"
```
Comment (lachlanjc) [inline app/controllers/events_controller.rb]:
```suggestion
      flash[:error] = 'We couldn’t find that event!'
```

**Review by lachlanjc — APPROVED** [review]

**Comment by tmb** [comment]
waiting on @lachlanjc to fix spacing and then we're good to merge!

**Comment by lachlanjc** [comment]
**THE DANGER ZONE**

Screen Shot 2019-05-31 at 12 39 33 PM

**Pull request merged**